app/controllers/main_controller.rb

```diff
@@ -9,3 +9,7 @@
 # Your controller actions go below this line.
 # -----------------------------------------------------------------------------
 
+MyApp.get "/" do
+  
+  erb :"views/welcome"
```

Comment:
Sinatra knows all views are in the **views/** folder, so you would just write `"welcome"` here. (This is the same as how it's been so far–just to be clear.)